src/policy/policy.cpp

```diff
@@ -35,7 +35,7 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
     // 182*dustRelayFee/1000 (in satoshis).
     // 546 satoshis at the default rate of 3000 sat/kvB.
     // A typical spendable segwit P2WPKH txout is 31 bytes big, and will
-    // need a CTxIn of at least 67 bytes to spend:
+    // need a CTxIn of at least 67 virtual bytes to spend:
```

Comment:
Yeah, this should be **vbytes** instead of bytes. The figure is also inaccurate, a P2WPKH input is [67.75 vbytes (for low r) or 68.0 vbytes for (high r)](https://bitcoin.stackexchange.com/a/100160/5406).

Reply:
@jonathanbier did you want to make the required updates here? Note that you'll need to [sqaush your commits](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits), and write a proper commit message etc.

Reply:
Ah, you might be looking for this Stack Exchange post then: https://bitcoin.stackexchange.com/a/41082/5406